lib/DxilPIXPasses/DxilDbgValueToDbgDeclare.cpp

```diff
@@ -704,40 +704,44 @@ void DxilDbgValueToDbgDeclare::handleDbgValue(
   auto* instruction = llvm::dyn_cast<llvm::Instruction>(V);
   if (instruction != nullptr) {
     instruction = instruction->getNextNode();
-    if (instruction != nullptr) {
+    if (instruction != nullptr && !llvm::isa<TerminatorInst>(instruction)) {
       // Drivers may crash if phi nodes aren't always at the top of a block,
       // so we must skip over them before inserting instructions.
       do {
         instruction = instruction->getNextNode();
-      } while (instruction != nullptr && llvm::isa<llvm::PHINode>(instruction));
+      } while (instruction != nullptr && 
+          llvm::isa<llvm::PHINode>(instruction) &&
+          !llvm::isa<TerminatorInst>(instruction));
```

Comment:
I don't think this is necessary as a PHI Node can't be a Terminator, it's harmless though.